src/main/java/org/isf/video/gui/PatientPhotoPanel.java

```diff
@@ -129,7 +144,7 @@ public PatientPhotoPanel(final PatientInsertExtended patientFrame, final Integer
 			jAttachPhotoButton.setMaximumSize(new Dimension(200, (int) jAttachPhotoButton.getPreferredSize().getHeight()));
 			jAttachPhotoButton.addActionListener(actionEvent -> {
 				JFileChooser fc = new JFileChooser();
-				String[] extensions = { "tif", "tiff", "jpg", "jpeg", "bmp", "png", "gif" };
+				String[] extensions = { "tif", "tiff", JPEG_FORMAT, "jpeg", "bmp", "png", "gif" };
```

Comment:
So the `jpg` extension is being removed?  Why?

Reply:
it is not removed, only replaced by (unnecessary?) constant

Reply:
See other comment.  The constant is mapped to "jpeg" (note the `e`) so the resulting `extensions` has `jpeg` twice (`JPEG_FORMAT, "jpeg"`) and no `jpg`.

Reply:
Ok, got it and fixed. Thanks!